providers/alibaba/docs/changelog.rst

```diff
@@ -26,6 +26,24 @@
 Changelog
 ---------
 
+3.0.2
+.....
+
+Misc
+~~~~
+
+* ``Add 'ti' to the RemoteLogIO read and upload methods (#48804)``
+* ``Rework remote task log handling for the structlog era. (#48491)``
```

Comment:
This should possibly be 3.1.0 now as is a new feature. 

Same on Azure,GCP and AWS providers

Reply:
It's a new feature of airdlow-core but it's a minor for the providor to support the new feature. Its a prespective we use many times in providers. Do you think otherwise?